.github/workflows/conbench_upload.yml

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Upload Benchmark Results
+on:
+  workflow_dispatch:
+    inputs:
+      run_id:
+        description: 'workflow run id to use the artifacts from'
+        required: true
+  workflow_run:
+    workflows: ["Linux Benchmark"]
+    types:
+      - completed
+
+permissions:
+  contents: read
+  actions: read
+  #TODO comment results to PR
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' || github.event_name == 'workflow_dispatch' }}
+    steps:
+
+    - name: 'Download artifacts'
+      id: 'download'
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const run_id = "${{ github.event.workflow_run.id || inputs.run_id }}";
```

Comment:
Does this run java script ? I am curious can someone create a PR and potentially download and js package to run stuff ?

Reply:
Yes this is js. The action provides an authenticated client (using $GITHUB_TOKEN)  of the [octokit.js](https://github.com/octokit/octokit.js) GitHub SDK/API client + some utility things.

octokit can be used locally with nodejs but needs some additional boiler plate to work (also the gha macros wont work ^^). Regarding PRs changes to this workflow will not take affect when in a PR, when this workflow is triggerd the version on main is always used.